robottelo/cli/hammer.py

```diff
@@ -33,7 +33,7 @@ def _csv_reader(output):
         data = data.encode('utf8')
     handler = StringIO(data)
 
-    for row in csv.reader(handler):
+    for row in csv.reader(handler):  # pragma: no cover
```

Comment:
Added `no cover` here because it is difficult to merge coverage reports for python 2 and 3 in final report